proto/composerapis/astria/composer/v1alpha1/generic_collector.proto

```diff
@@ -0,0 +1,19 @@
+syntax = 'proto3';
+
+package astria.composer.v1alpha1;
+
+import "google/protobuf/empty.proto";
+
+message SequenceAction {
+  string rollup_id = 1;
```

Comment:
`rollup_name` (ie plaintext string) or the 32 byte `rollup_id`? should probably be `rollup_id` to match other APIs, but in that case, you can just use the `SequenceAction` defined in sequencerapis

Reply:
i was actually think we don't want them to specify any native_fee_asset because composer is technically a gas station in this case. we want to pay the fee for sending the sequence actions.

Reply:
that makes sense!